src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/Models/ReindexJobRecord.cs

```diff
@@ -115,13 +115,13 @@ protected ReindexJobRecord()
         public IReadOnlyCollection<string> TargetResourceTypes { get; private set; } = new List<string>();
 
         [JsonIgnore]
-        public int PercentComplete
+        public decimal PercentComplete
```

Comment:
>PercentComplete [](http://example.com/codeflow?start=23&length=15)

another nit:
Can we actually move it to `ToParametersResourceElement` function?
It's only used there, so I don't see much value in having this property here.

Reply:
Done